benefits/eligibility/templates/eligibility/start--mst-courtesy-card.html

```diff
@@ -2,7 +2,7 @@
 {% load i18n %}
 
 {% block page-title %}
-  {% translate "Courtesy Card information" %}
+  {% translate "Courtesy Card overview" %}
```

Comment:
In the spreadsheet, specifically the `forTranslation` sheet, there's no instance of `Courtesy Card overview`.

![image](https://github.com/cal-itp/benefits/assets/25497886/ec434b5b-8319-4cab-ba56-4e094153257c)

Should we have sent "Courtesy Card overview" to the translation service, or are we supposed to use "Agency card overview" here?

Reply:
^ Question for you @indexing

Reply:
^ Question for you @indexing

Reply:
I think it's missing from the `forTranslation` tab

Reply:
The bigger point here is that devs should be able to take exactly what's in the spreadsheet and transfer it to the .PO files. Devs should not have to make judgement calls on what the "correct copy" to use is.

Reply:
Definitely

Reply:
> In the spreadsheet, specifically the `forTranslation` sheet, there's no instance of `Courtesy Card overview`.

> Should we have sent "Courtesy Card overview" to the translation service, or are we supposed to use "Agency card overview" here?

@angela-tran When I included the page titles in the copy master, I removed any agency-specific strings from the titles to reduce the number of custom strings that required variables or translation. "Courtesy Card" is terminology specific to MST, so the title is now "Agency card overview" so it can be used with any agency. (SBMTD will use the same page title as MST for this view.)

All agency-specific strings are in the tab for that agency. So "Courtesy Card" will be used in the main copy of each view for MST.

So the spreadsheet should be accurate, but please let me know if you see any other anomalies.

Reply:
@indexing Got it. Just to give a little extra background, "agency cards" is just a term we made up for easier communication within the team; I'm not sure if a user would immediately know what that is. But if you think it is fine and makes managing our page titles easier, that sounds good to me.

@machikoyasuda Based on Andy's comment, specifically
> ... so the title is now "Agency card overview" so it can be used with any agency. (SBMTD will use the same page title as MST for this view.)

this line should change into:
```suggestion
  {% translate "Agency card overview" %}
```